wire-ios/Wire-iOS.xcodeproj/project.pbxproj

```diff
@@ -1567,8 +1567,6 @@
 		EEE25B4E29719D0B0008B894 /* ZipArchive.xcframework in Embed Frameworks */ = {isa = PBXBuildFile; fileRef = EEE25B4C29719D0B0008B894 /* ZipArchive.xcframework */; settings = {ATTRIBUTES = (CodeSignOnCopy, RemoveHeadersOnCopy, ); }; };
 		EEE25B5029719D2F0008B894 /* avs.xcframework in Frameworks */ = {isa = PBXBuildFile; fileRef = EEE25B4F29719D2F0008B894 /* avs.xcframework */; };
 		EEE25B5129719D2F0008B894 /* avs.xcframework in Embed Frameworks */ = {isa = PBXBuildFile; fileRef = EEE25B4F29719D2F0008B894 /* avs.xcframework */; settings = {ATTRIBUTES = (CodeSignOnCopy, RemoveHeadersOnCopy, ); }; };
-		EEE25B5329719D540008B894 /* libPhoneNumberiOS.xcframework in Frameworks */ = {isa = PBXBuildFile; fileRef = EEE25B5229719D540008B894 /* libPhoneNumberiOS.xcframework */; };
```

Comment:
Not sure where to comment, so I do it here to keep the conversation:
The configuration resource .xcconfig contain a flag `WIRESTAN`, that should be removed from build assets, cc @netbe .
`// WIRESTAN - Enabled the fake country in the phone number selection`

Reply:
Can I address this on an follow up PR?

Reply:
@agisilaos yes create a ticket in jira and follow up :)